.github/scripts/cmake.sh

```diff
@@ -85,40 +80,24 @@ fi
 popd
 echo '::endgroup::'
 
-echo '::group::Build and run project that uses Faster-RCNN'
-pushd test/tracing/frcnn/build
-
-cmake .. -DTorch_DIR="${Torch_DIR}" -DWITH_CUDA="${WITH_CUDA}" \
-  -DCMAKE_PREFIX_PATH="${CONDA_PREFIX}" \
-  -DCMAKE_FIND_FRAMEWORK=NEVER
-if [[ $OS_TYPE == windows ]]; then
-  "${PACKAGING_DIR}/windows/internal/vc_env_helper.bat" "${PACKAGING_DIR}/windows/internal/build_frcnn.bat" $JOBS
-  cd Release
-  cp ../fasterrcnn_resnet50_fpn.pt .
-else
-  make -j$JOBS
-fi
-
-./test_frcnn_tracing
-
-popd
-echo '::endgroup::'
-
 echo '::group::Build and run C++ example'
-pushd examples/cpp/hello_world/build
+pushd examples/cpp/build
 
 cmake .. -DTorch_DIR="${Torch_DIR}" \
   -DCMAKE_PREFIX_PATH="${CONDA_PREFIX}" \
-  -DCMAKE_FIND_FRAMEWORK=NEVER
+  -DCMAKE_FIND_FRAMEWORK=NEVER \
+  -DUSE_TORCHVISION=ON  # Needed for faster-rcnn since it's using torchvision ops like NMS.
```

Comment:
Some scripted models like Faster-RCNN rely on torchvision ops, so for those we need to link to `libtorchvision`. But some other models like resnet do not need `libtorchvision`.

Prior to this PR, the example's `CmakeLists.txt` would  always link to `libtorchvision` even if it was unnecessary. I added a `USE_TORCHVISION` option to let use decide whether they want to link against it or not. Here, we want to test the RCNN model, so we have to pass `USE_TORCHVISION=ON`.